src/components/TwoOptionsToggle.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { COLORS } from '../styles/themes';
+
+/**
+ * @param {{
+ *   options: Map<'first'|'second', { value: string; onTap: Function; }>;
+ *   defaultOption: 'first'|'second';
+ * }}
+ */
+export const TwoOptionsToggle = ({ options, defaultOption }) => {
+  const [currOption, setCurrOption] = useState(defaultOption);
+  const isActive = (option) => (componentOption) => option === componentOption;
+  const onTap = (option) => {
+    setCurrOption(option);
+    // Execute the callback assigned to the option
+    options[option].onTap();
+  };
+  return (
+    <View style={[styles.wrapper]}>
```

Comment:
suggestion(non-blocking): I think it would look great if we added a transition on the wrapper to make the toggle actually move between the options.

Reply:
Definitely! I will open a PR of improvement and we add it latter.

Reply:
Here the issue: https://github.com/HathorNetwork/hathor-wallet-mobile/issues/482